Formula/mothur.rb

```diff
@@ -0,0 +1,36 @@
+class Mothur < Formula
+  desc "Analyse microbial ecology data"
+  homepage "https://www.mothur.org/"
+  url "https://github.com/mothur/mothur/archive/v1.39.5.tar.gz"
+  sha256 "9f1cd691e9631a2ab7647b19eb59cd21ea643f29b22cde73d7f343372dfee342"
+  head "https://github.com/mothur/mothur.git"
+
+  depends_on "boost"
+
+  def install
+    boost = Formula["boost"]
+    system "make", "BOOST_LIBRARY_DIR=#{boost.opt_lib}",
+                   "BOOST_INCLUDE_DIR=#{boost.opt_include}"
+    bin.install "mothur", "uchime"
```

Comment:
What's the status on getting a `make install`?

Reply:
`make install` made it to master, dunno yet if they're gonna have a patch release or if it will be included in the next minor release

Reply:
Can you apply the patch in the formula using `patch do`?

Reply:
done but fails audit. should I submit the patch to https://github.com/Homebrew/formula-patches or is a gist fine?

Reply:
No worries about the audit, since that only applies to --new-formula and the patch has been merged upstream.

You'll want to scope it to a `stable do` block, or we can just drop `head` since it's probably not needed anyway.

>should I submit the patch to https://github.com/Homebrew/formula-patches

yes please

Reply:
@HadrienG there's a minor issue with naming that needs to be fixed too: https://github.com/mothur/mothur/pull/385#issuecomment-334994511